patientsearch/src/js/components/PatientListTable.js

```diff
@@ -259,7 +259,8 @@ export default function PatientListTable(props) {
         setOpenLoadingModal(false);
         return false;
       }
-      if (!results[1] || (results[1] && !results[1].valid)) {
+      if (!results[1] ||
+         (results[1] && (parseInt(results[1].access_expires_in) <= 0 || parseInt(results[1].refresh_expires_in) <= 0))) {
         //invalid token, force logout
         console.log("Logging out...")
         handleLogout();
```

Comment:
Instead of logging out when the token is absent/invalid, can you redirect to `/`?

Since `/` is a protected endpoint, the backend will request a new Access Token from Keycloak if able, else prompt a user to log in again.

For context, `/logout` currently logs a user out of both cosri-patientsearch AND Keycloak, even though there may still be a valid Keycloak session to request a new Access Token from.

Reply:
thanks, fixed